Fix one more invalid escape sequence that makes it into the package
---
 llvm/tools/opt-viewer/opt-viewer.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff -pru llvm-project.orig/llvm/tools/opt-viewer/opt-viewer.py llvm-project/llvm/tools/opt-viewer/opt-viewer.py
--- llvm-project.orig/llvm/tools/opt-viewer/opt-viewer.py	2024-12-04 18:23:16.754546940 -0500
+++ llvm-project/llvm/tools/opt-viewer/opt-viewer.py	2024-12-04 19:10:24.893308338 -0500
@@ -115,7 +115,7 @@ class SourceFileRenderer:
         # Column is the number of characters *including* tabs, keep those and
         # replace everything else with spaces.
         indent = line[:max(r.Column, 1) - 1]
-        indent = re.sub('\S', ' ', indent)
+        indent = re.sub(r'\S', ' ', indent)
 
         # Create expanded message and link if we have a multiline message.
         lines = r.message.split('\n')
-- 

